<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>896</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>JuKuZ Livestreaming Regie</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1441</width>
      <height>880</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="CamBox" name="groupBox_4">
       <property name="minimumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="CamBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="CamBox" name="groupBox_2">
       <property name="minimumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="CamBox" name="groupBox_6">
       <property name="minimumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="CamBox" name="groupBox_5">
       <property name="minimumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="CamBox" name="groupBox_3">
       <property name="minimumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>388</width>
         <height>284</height>
        </size>
       </property>
       <property name="title">
        <string>GroupBox</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="3">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="minimumSize">
          <size>
           <width>650</width>
           <height>380</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>644</width>
           <height>380</height>
          </size>
         </property>
         <property name="title">
          <string>Ausgabevorschau</string>
         </property>
         <widget class="QGraphicsView" name="videoBox">
          <property name="geometry">
           <rect>
            <x>5</x>
            <y>20</y>
            <width>640</width>
            <height>360</height>
           </rect>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="lineWidth">
           <number>0</number>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="minimumSize">
          <size>
           <width>650</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>650</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>GroupBox</string>
         </property>
         <widget class="QWidget" name="gridLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>631</width>
            <height>121</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QCheckBox" name="logoButton">
             <property name="text">
              <string>SenderLogo(s)</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="logoFileLineEdit"/>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="logoFileSelectButton">
             <property name="text">
              <string>Ausw√§hlen ...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="textButton">
             <property name="text">
              <string>Text aktivieren</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="textEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>16</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CamBox</class>
   <extends>QGroupBox</extends>
   <header>cambox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
